server/webhook_worker.go

```diff
@@ -4,8 +4,11 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
 	"time"
 
+	"github.com/andygrunwald/go-jira"
 	"github.com/mattermost/mattermost-plugin-jira/server/utils"
 	"github.com/mattermost/mattermost-plugin-jira/server/utils/types"
```

Comment:
CI is having issues with the error `File is not 'goimports'-ed with -local`. This is happening because we are grouping external packages with our own internally defined ones here. We need to have a new line in between these imports:

```suggestion
	"github.com/andygrunwald/go-jira"

	"github.com/mattermost/mattermost-plugin-jira/server/utils"
	"github.com/mattermost/mattermost-plugin-jira/server/utils/types"
```

After this change is made, it results in three properly separated groups of imports:
- built-in Go packages first (`fmt`, `time` etc.)
- external packages (usually prefixed with `github.com`)
- packages defined in this project (contains `mattermost-plugin-jira`)

Reply:
@mickmister Thankyou , this helped a lot